### ISSUE TITLE
Usability enhancement: do not distinguish input data units only by colour

### ISSUE THREAD

**Issue by jindrichmynarz** [linkedpipes/etl#394]
In the latest develop version 344846a472fe14e5456e9443278e9cb7862abf1e the input data units for configuration and data are distinguished only by colour:

<img width="167" alt="screen shot 2017-06-01 at 12 31 32" src="https://cloud.githubusercontent.com/assets/198642/26676016/5c3b595a-46c6-11e7-9466-dab23eccd02f.png">

I can only guess that violet is for configuration and green for data. Is this already documented?

Users may be confused about choosing the data unit. Some users are colour blind, so they cannot distinguish the input data units (except by memorizing their positions). It would be helpful if they were distinguished not only by colour, but by other means too, e.g., by shape.

**Comment by jakubklimek** [comment]
Different shape is not a bad idea. The question is whether the library used will allow this easily.

Since this is about two days old in develop, it is not documented and will not be in the near future unyil the feature stabilizes.

**Comment by jindrichmynarz** [comment]
This issue is difficult even for users without vision problems, especially for components that accept multiple inputs. In that case, the only way how to figure which input to use is to drag an edge from an output port and selected the appropriate component and input port from the pop-up.

While using different shapes may be difficult to implement, please at least add a title that displays on hover.

**Comment by jakubklimek** [comment]
Actually, when a component has multiple data inputs, their labels are shown:
![image](https://user-images.githubusercontent.com/718827/28156166-e8e9b728-67b1-11e7-8326-ab74da3921af.png)
Only when there is one data input and one configuration input, the labels are not shown, then it is distinguished by color. This issue is about distinguishing it also by shape. Maybe we could also have a Personalization option to show the labels always, which, however, results in a messy looking pipeline.

I would prefer not to do "on hover" things due to their unavailability on touch devices. If something needs to be communicated to the user, it should be either explicitly shown, or a switch for this should be available.

**Comment by jindrichmynarz** [comment]
You're right. I noticed that some components now show labels of the input ports. However, not all components do that:

<img width="294" alt="screen shot 2017-07-13 at 10 03 09" src="https://user-images.githubusercontent.com/198642/28156337-975e9de6-67b2-11e7-9215-d5b8eff4ab80.png">

Showing titles on hover is not ideal, but it is better than nothing, especially considering that a users on touch devices would be a tiny minority for a tool like LP-ETL.
